src/python/grpcio_tests/tests/unit/_signal_handling_test.py

```diff
@@ -208,7 +208,6 @@ def testStreamingHandlerWithException(self):
                 self._handler.await_connected_client()
                 client.send_signal(signal.SIGINT)
                 client.wait()
-                print(_read_stream(client_stderr))
```

Comment:
Is this intentional?

Reply:
Yes this was the only line being output in stdout. all other tests have 0 stdout

Reply:
Instead of deleting, can we capture this to a variable and conditionally log in case the assertion on the next line fires?